esmvaltool/config-references.yml

```diff
@@ -482,7 +482,7 @@ references:
   wenzel14jgr: "Wenzel et al., J. Geophys. Res. Biogeosci., 119(5), doi:2013JG002591, 2014."
   williams09climdyn: "Williams and Webb, Clim. Dynam., 33, 141-157, doi:10.1007/s00382-008-0443-1, 2009."
 
-observation references:
+observation_references:
```

Comment:
This is more of a general question, but why is there a separate list for observational references? Wouldn't it make more sense to have a single list of references?

Reply:
obs people are `special` :grin: Beats me, man - @mattiarighi ?

Reply:
The observation_references have the dataset name associated to a paper, a link, a source, or whatever the data are documented.
It is fine to have a single list, as long as the key for the obs remains the dataset name.

Reply:
Let's make it one list then. That will work nicer with provenance collection, because then you can also add observations as references (or we may automate this at some point).

Reply:
Btw., that list should be cleaned, updated and some entries removed since the datasets are not yet available (see [status](https://github.com/ESMValGroup/ESMValTool/issues/232#issuecomment-457246246)). 

And I would also remove the e-mail addresses of all non-core developers.

Shall I do this?

Reply:
I'm not sure if it helps much to remove the email addresses from that list. Email addresses are included in every git commit message anyway, but it's up to you.